packages/react-components/src/components/styles/Captions.style.ts

```diff
@@ -42,5 +42,6 @@ export const captionContainerClassName = mergeStyles({
  */
 export const captionsBannerClassName = mergeStyles({
   height: _pxToRem(100),
-  overflow: 'scroll'
+  overflowY: 'scroll',
```

Comment:
We can probably simplify to 

overflow: 'hidden scroll'

![image](https://user-images.githubusercontent.com/79329532/231628301-814be63d-2993-498f-b1e9-c02dbd402d2b.png)

Reply:
<img width="910" alt="Screenshot 2023-04-13 at 9 56 13 AM" src="https://user-images.githubusercontent.com/96077406/231831104-c0666707-68cb-4a83-b5ac-e2478aa6d731.png">
Tried it and looks like IStyle doesn't support 'hidden scroll'

Reply:
cool then lets keep it separate